### ISSUE TITLE
Accessibility of login page

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1112]
There are simple things that could be done to improve accessibility of main login screen:

- [ ] There's no alt text on Xerte or Apereo logos on header or any of images on footer bar
- [ ] 'Username' & 'password' strings need to be marked up as labels for the login input fields
- [ ] More meaningful text on the 'Show me!' button
- [ ] Use headings, regions and roles on the different areas?

The look of the login page could probably have a bit of a revamp too. New background image etc.
